bccm_core/lib/src/utils/images.dart

```diff
@@ -74,9 +74,11 @@ ImageProvider<Object> networkImageWithRetryAndResize({
   return ResizeImage.resizeIfNeeded(
     null,
     cacheHeight,
-    NetworkImageWithRetry(
+    ExtendedNetworkImageProvider(
       imageUrl,
       headers: const {'Keep-Alive': 'timeout=20, max=5'},
+      retries: 3,
+      timeRetry: const Duration(milliseconds: 100),
```

Comment:
These are the default values of the provider, but I think it is useful to be explicit here to make it easier to understand